### PR TITLE
- add parg/1.0.2

### PR DESCRIPTION
**Pull request by SSE4**
Specify library name and version:  **parg/1.0.2**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/parg/all/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.1)
+project(cmake_wrapper)
+
+include("conanbuildinfo.cmake")
+conan_basic_setup()
```
Comment (SSE4) [inline recipes/parg/all/CMakeLists.txt:5]:
```suggestion
conan_basic_setup()
set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
```

File: recipes/parg/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        cmake.build()
+
+    def package(self):
+        self.copy("COPYING", dst="licenses", src=self._source_subfolder)
+        self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
+        cmake = self._configure_cmake()
+        cmake.install()
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        tools.rmdir(os.path.join(self.package_folder, "share"))
```
Comment (SpaceIm) [inline recipes/parg/all/conanfile.py:61]:
```suggestion
```

File: recipes/parg/all/test_package/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
```
Comment (SpaceIm) [inline recipes/parg/all/test_package/CMakeLists.txt:2]:
```suggestion
project(test_package C)
```

File: recipes/parg/all/test_package/CMakeLists.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
```
Comment (SpaceIm) [inline recipes/parg/all/test_package/CMakeLists.txt:5]:
```suggestion
conan_basic_setup(TARGETS)
```

File: recipes/parg/all/conanfile.py [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+        self._cmake = cmake
+        return self._cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy("COPYING", dst="licenses", src=self._source_subfolder)
+        self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
```
Comment (SpaceIm) [inline recipes/parg/all/conanfile.py:56]:
```suggestion
```

File: recipes/parg/all/test_package/CMakeLists.txt (hunk 3 above)
Comment (SSE4) [reply recipes/parg/all/test_package/CMakeLists.txt:2]: I am curious, does it have any actual effect?

Comment (SpaceIm) [reply recipes/parg/all/test_package/CMakeLists.txt:2]: Yes, it doesn't try to find or call a C++ compiler or linker (nice to properly test C++ library with C API and a C test package). Here it's not very important.

File: recipes/parg/all/test_package/CMakeLists.txt (hunk 4 above)
Comment (SSE4) [reply recipes/parg/all/test_package/CMakeLists.txt:5]: not sure if it's important as well, as we use neither `CONAN_LIBS` nor `CONAN_PKG::` here.
`conan_basic_setup` is only needed to set correct flags.

Comment (SpaceIm) [reply recipes/parg/all/test_package/CMakeLists.txt:5]: it's important to really test your CMake target.

**Review by SpaceIm — APPROVED** [review]

File: recipes/parg/all/test_package/conanfile.py [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self.settings):
+            bin_path = os.path.join("bin", "test_package")
+            self.run("%s -h" % bin_path, run_environment=True)
```
Comment (prince-chrismc) [inline recipes/parg/all/test_package/conanfile.py:17]: Do we really need to parse? Can we simplify the text package?

**Review by prince-chrismc — APPROVED** [review]

File: recipes/parg/all/test_package/conanfile.py (hunk 6 above)
Comment (SpaceIm) [reply recipes/parg/all/test_package/conanfile.py:17]: IMOO, test package is simple enough

Comment (SSE4) [reply recipes/parg/all/test_package/conanfile.py:17]: yes, I am not sure what do you mean, if you can illustrate with some example, would be nice

**Review by jgsogo — APPROVED** [review]

File: recipes/parg/all/test_package/test_package.c [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+		default:
+			printf("error: unhandled option -%c\n", c);
+			return EXIT_FAILURE;
+			break;
+		}
+	}
+
+	for (c = ps.optind; c < argc; ++c) {
+		printf("nonoption '%s'\n", argv[c]);
+	}
```
Comment (prince-chrismc) [inline recipes/parg/all/test_package/test_package.c:47]:
```suggestion
```

**Pull request merged**